addons/web/static/src/js/views/form/form_controller.js

```diff
@@ -218,7 +235,32 @@ var FormController = BasicController.extend({
     //--------------------------------------------------------------------------
     // Private
     //--------------------------------------------------------------------------
-
+    /**
+     * Assign on the buttons save and discard additionnal behavior to facilitate the work of the users doing input only using the keyboard
```

Comment:
80 characters... it may be annoying to you, but i really prefer being able to read the doc

Reply:
done